repondeur/zam_repondeur/fetch/senat/amendements/parse.py

```diff
@@ -1,84 +1,85 @@
 import re
-from typing import Optional
+from typing import Dict, Optional, Tuple, cast
 from urllib.parse import urlparse
 
 from zam_repondeur.clean import clean_html
 from zam_repondeur.fetch.dates import parse_date
 from zam_repondeur.fetch.division import _parse_subdiv
-from zam_repondeur.fetch.models import Amendement
+from zam_repondeur.models import (
+    DBSession,
+    Article,
+    Amendement,
+    Lecture,
+    get_one_or_create,
+)
 
 
-def parse_from_csv(row: dict, session: str, num_texte: int, organe: str) -> Amendement:
-    num, rectif = Amendement.parse_num(row["Numéro "])
-    subdiv_type, subdiv_num, subdiv_mult, subdiv_pos = _parse_subdiv(
-        row["Subdivision "]
+def parse_from_csv(row: dict, lecture: Lecture) -> Tuple[Amendement, bool]:
+    subdiv = _parse_subdiv(row["Subdivision "])
+    article, created = get_one_or_create(  # type: ignore
+        DBSession,
+        Article,
+        type=subdiv.type_,
+        num=subdiv.num,
+        mult=subdiv.mult,
+        pos=subdiv.pos,
```

Comment:
Same problem here, looks like an Article with the same number from a previous Lecture from would be reused.